components/pagination.tsx

```diff
@@ -0,0 +1,69 @@
+"use client"
+
+import { useRouter, useSearchParams } from "next/navigation"
+import { LIMIT, MIN_PAGE_NUMBER } from "@/constants/pagination"
+
+interface PaginationProps {
+  totalCount: number
+}
+
+export function Pagination({ totalCount }: PaginationProps) {
+  const router = useRouter()
+  const searchParams = useSearchParams()
+
+  const currentPage = Number(searchParams.get("page")) ?? 1
+  const totalPages = Math.ceil(totalCount / LIMIT)
+
+  const hasPrevPage = currentPage > MIN_PAGE_NUMBER
+  const hasNextPage = currentPage < totalPages
+
+  const showPlayersFrom = LIMIT * (currentPage - 1) + 1
+  const showPlayersTo = LIMIT * currentPage
```

Comment:
Would be nice to change the reference to players as this component is generic, like this
```suggestion
  const showItemFrom = LIMIT * (currentPage - 1) + 1
  const showItemTo = LIMIT * currentPage
```

Reply:
@romansharapov19, looks like `showItemFrom` isn't being used.

<img width="873" alt="image" src="https://github.com/berTrindade/video-player-manager/assets/13338262/02735569-1b01-4d1a-b628-12217adb1c56">